src/qforte/ucc/uccnpqe.py

```diff
@@ -167,7 +170,12 @@ def print_summary_banner(self):
         print('Number of non-zero res element evaluations:  ', int(self._res_vec_evals)*self._n_nonzero_params)
 
     def solve(self):
```

Comment:
Again, I wonder if this belongs in `uccpqeabc.py`.

Reply:
I agree again. This one did not occur to me. I will do the transfer of code and push.